### PR TITLE
Clean up block master service handler

### PR DESCRIPTION
**Pull request by calvinjia**
Removes some unnecessary typing and also fixes the name of some methods so we collect the correct metrics.

### CODE REVIEW

**Comment by calvinjia** [comment]
@ZacBlanco Good call, I've updated the format for the methods which do not require a multi-line closure. For the multi-line one I kept it as is since I think its more readable.

File: core/server/master/src/main/java/alluxio/master/block/BlockMasterClientServiceHandler.java [hunk 1]
```diff
@@ -62,113 +62,95 @@ public BlockMasterClientServiceHandler(BlockMaster blockMaster) {
...
-
-    RpcUtils.call(LOG, (RpcUtils.RpcCallableThrowsIOException<GetBlockInfoPResponse>) () -> {
-
-      return GetBlockInfoPResponse.newBuilder()
-          .setBlockInfo(GrpcUtils.toProto(mBlockMaster.getBlockInfo(blockId))).build();
-    }, "CreateFile", "blockId=%s, options=%s", responseObserver, blockId, options);
+    RpcUtils.call(LOG,
+        () -> GetBlockInfoPResponse.newBuilder()
+            .setBlockInfo(GrpcUtils.toProto(mBlockMaster.getBlockInfo(blockId))).build(),
+        "GetBlockInfo", "blockId=%s, options=%s", responseObserver, blockId, options);
```
Comment (apc999) [inline core/server/master/src/main/java/alluxio/master/block/BlockMasterClientServiceHandler.java]: getBlockInfo?

Comment (calvinjia) [reply core/server/master/src/main/java/alluxio/master/block/BlockMasterClientServiceHandler.java]: I changed it all to capitalized to be consistent with our other logging.

**Comment by apc999** [comment]
alluxio-bot, merge this please

**Comment by apc999** [comment]
@calvinjia I view this actually a bug fix .  do we target this to branch 2.1?

**Comment by calvinjia** [comment]
@apc999 that makes sense

**Comment by calvinjia** [comment]
alluxio-bot, cherry-pick this to branch-2.1 please.

**Pull request merged**